### ISSUE TITLE
From the paper Uncertainty - minimal Query Selection Uses Encoder Outputs aka $\hat{X}$ But I dont see it in code ?

### ISSUE THREAD

**Issue by r4hul77** [lyuwenyu/RT-DETR#357]
Hello Firstly Thanks for rt-detr, I was really curious about how the query selection where query selection is done and I believe it is done in vfl loss. As per my intuition the query selection has to be done using the encoder outputs $\hat{X}$, but the vfl loss just uses decoder outputs aka "pred_boxes" and "pred_logits". Can you please tell me what am I missing from the code ?

**Comment by Reaidu** [comment]
@lyuwenyu I also don't understand where the minimum query is mentioned in the code，thanks~
